### ISSUE TITLE
Unnecessary DNS servers in /etc/resolv.conf file

### ISSUE THREAD

**Issue by brainfucksec** [BlackArch/torctl#23]
Is there a particular reason why all these DNS servers are put in the `/etc/resolv.conf` file?
Only the `localhost` DNS server is needed to set the Transparent Proxy.

I know it is not necessary for you to link it, but I am including the source as a reference for this issue, see:

https://gitlab.torproject.org/legacy/trac/-/wikis/doc/TransparentProxy#local-redirection-through-tor

**Comment by cristiancmoises** [comment]
Exactly. And one of the most secure DNS in the world are: 9.9.9.9 and 149.112.112.112   the QUAD9 ipv4.
I dont like the '8.8.8.8' in the script because: GOOGLE!
